src/main/java/org/folio/service/pieces/flows/DefaultPieceFlowsValidator.java

```diff
@@ -56,6 +58,16 @@ public static boolean isCreateHoldingForPiecePossible(Piece pieceToCreate, Compo
                         && PoLineCommonUtil.isHoldingUpdateRequiredForPhysical(originPoLine));
   }
 
+  public static List<Error> isDisplayOnHoldingsNotConsistent(Piece piece) {
```

Comment:
first I read the code, I consfused, method is more like return boolean. Suggestion createErrorIfDisplayHodlingNotConsisted..
combineErrorIfDisplayHodlingNotConsisted..

var combingError = createErrorIfDisplayHodlingNotConsisted..

Reply:
other similar methods also have 'validate' prefix, so renemed to validateDisplayOnHoldingsConsistency